### PR TITLE
Handle assembly already loaded errors in deploy stress test script

### PR DESCRIPTION
**Pull request by benbp**
This issue has been showing up recently: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1103685&view=logs&j=38a05975-7697-5434-099a-d7b3d4012883&t=b1e71580-9408-50be-13a0-b0783b8cb008&l=57

```
Installing module az with min version 6.4.0 from https://www.powershellgallery.com/api/v2
Using module Az with version 6.4.0.
Import-Module : Assembly with same name is already loaded
At /home/vsts/.local/share/powershell/Modules/Az/6.4.0/Az.psm1:113 char:1
+ Import-Module Az.Aks -RequiredVersion 2.4.0 -Global
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (:) [Import-Module], FileLoadException
+ FullyQualifiedErrorId : FormatXmlUpdateException,Microsoft.PowerShell.Commands.ImportModuleCommand

```

### CODE REVIEW

File: eng/common/scripts/stress-testing/deploy-stress-tests.ps1 [hunk 1]
```diff
@@ -21,8 +21,8 @@ $ErrorActionPreference = 'Stop'
 $FailedCommands = New-Object Collections.Generic.List[hashtable]
 
 . (Join-Path $PSScriptRoot "../Helpers" PSModule-Helpers.ps1)
-Install-ModuleIfNotInstalled "powershell-yaml" "0.4.1" | Import-Module
-Install-ModuleIfNotInstalled "az" "6.4.0" | Import-Module
+Install-ModuleIfNotInstalled "powershell-yaml" "0.4.1" | Import-Module -ErrorAction Continue
```
Comment (weshaggard) [inline eng/common/scripts/stress-testing/deploy-stress-tests.ps1:24]: If this is a common error we should consider adding a helper function for importing.

**Comment by weshaggard** [comment]
Is this a real problem? We should only be importing it if it isn't already loaded.

**Comment by benbp** [comment]
@weshaggard I'm not quite sure what's going on here, or why it seems to be only affecting this pipeline and not ALL pipelines that use this. I thought of this as a short term workaround but I really would like to understand the root issue here.

**Comment by benbp** [comment]
@weshaggard turns out it doesn't work anyway: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1108561&view=logs&j=38a05975-7697-5434-099a-d7b3d4012883&t=b1e71580-9408-50be-13a0-b0783b8cb008&l=58

**Comment by weshaggard** [comment]
I'm not sure why it isn't correctly detecting the loaded module but it looks like you aren't using this in a normal powershell step but instead as part of some Azure CLI step is there a reason for that?

**Comment by benbp** [comment]
> I'm not sure why it isn't correctly detecting the loaded module but it looks like you aren't using this in a normal powershell step but instead as part of some Azure CLI step is there a reason for that?

Now I'm not so sure why I even added this as a dependency at all. We don't use the Az cmdlets in this script...

**Review by weshaggard — APPROVED** [review]

**Pull request merged**